tests/test-services/src/terminal/manager.spec.ts

```diff
@@ -124,7 +126,7 @@ describe('terminal', () => {
         let called = false;
         session = await manager.startNew();
         manager.runningChanged.connect((sender, args) => {
-          expect(session.isDisposed).to.equal(false);
+          expect(session.isDisposed).to.equal(true);
```

Comment:
Note to self: this may need to be changed back to false after my recent manager updates.